.travis.yml

```diff
@@ -41,6 +41,12 @@ matrix:
         - DESC="checkstyle and sevntu-checkstyle"
         - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dcobertura.skip=true"
 
+    # jacoco (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="jacoco"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -Pjacoco-check -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true"
```

Comment:
Aren't we swapping cobertura entirely for jacoco? I don't see why we would keep the cobertura since it is abandoned.
Why does this need a profile? cobertura didn't have a profile.

Reply:
We need to keep the cobrature for a while. But yes, soon it will be replaced. 
Now we need to do some regression tests.

Reply:
> We need to keep the cobratura for a while.

Why?

Reply:
We will switch to jacoco only when we are ready. We need to test it by some time and accept jacoco in multiple steps.

If jacoco make problems, I will switch it off.

Cobertura stay as primary check till final fix for jacoco,

Reply:
@romani Then please review the PR and specify what you want done in it.

Reply:
@pbludov , as you use "verify", please disable all other static analysers, review profile "no-validations" at https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L1486

@rnveach, as update to jacoco is hard to do in one PR, we should do this in few PRs.
PRs:
1) initial jacoco coverage for coverage as-as
2)-X) fixes for non 100 coverages in files (except for grammar and UI)
Y) update web site to have jacoco report, xdoc for development requirements, update wiki ....
Z) let cobetura and jacoco run together for some time too see how it is working in PRs and catch all contributors problems and concerns.
Final: disable cobertura.

Reply:
> verify -Pjacoco-check -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true

"verify" + not all skips.
@pbludov , please take a look at link I provided and make all skips except for UT(required for jacoco).
In build log there should be no other plugin running, to speed up the build process.